lib/Dialect/MSFT/ExportQuartusTcl.cpp

```diff
@@ -46,21 +46,35 @@ struct TclOutputState {
 };
 } // anonymous namespace
 
+void emitInnerRefPart(TclOutputState &s, Operation *op) {
+  // Extract the name from an InstanceOp or other op with a "name".
+  StringAttr nameAttr;
+  if (auto instOp = dyn_cast<msft::InstanceOp>(op))
```

Comment:
Does the lowered hw.instance have `instanceName` as a symbol? Or does `sym_name` get propagated instead. They're the same for now, but that may change.